src/pages/ApplyPage.tsx

```diff
@@ -99,6 +102,9 @@ function ApplyPage() {
           data-aos="fade-right"
           data-aos-duration="1000"
         />
+        <S.LottieDiv>
+          <Lottie options={applyPageTeamLottieOptions} />
+        </S.LottieDiv>
```

Comment:
From here you can add a `LottieDiv` component in `ApplyPage.styles.ts` to add css rules for the position and size.